english/wakeup/wake.md

```diff
@@ -0,0 +1 @@
+i woke up in the middle of the night, i ask here what time is it, she looked at me and say there is no time here in this world just enjoy the eternity this how every thing begin.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.